chart/templates/job-db-migrate.yaml

```diff
@@ -25,13 +25,13 @@ spec:
       affinity:
         podAffinity:
           requiredDuringSchedulingIgnoredDuringExecution:
-          - labelSelector:
-              matchExpressions:
-                - key: app.kubernetes.io/part-of
-                  operator: In
-                  values:
-                    - rails
-            topologyKey: kubernetes.io/hostname
+            - labelSelector:
```

Comment:
isn't this technically putting it on the wrong level?
if you look at `kubectl get pod -o yaml` the following output shows:
```yaml
spec:
  affinity:
    podAffinity:
      requiredDuringSchedulingIgnoredDuringExecution:
      - labelSelector:
```

Reply:
I think its fine! This change is equivalent to a change like going from...

```yaml
myList:
- 1
- 2
```

to...

```yaml
myList:
  - 1
  - 2
```

The YAML is formatted differently, but represents the same data. `kubectl get pod -o yaml` may use a YAML formatter that emits YAML without spacing. This Helm chart has consistently used two spaces before the `-` character for each element in sequences/lists though.